packages/shared/styles/components/molecules/SfPagination.scss

```diff
@@ -4,32 +4,34 @@
   align-items: center;
   &__item {
     --link-text-decoration: none;
-    --link-color: var(--c-text-muted);
+    color: var(--pagination-item-color, var(--c-gray-variant));
     width: var(--pagination-item-width);
     height: var(--pagination-item-height);
     margin: var(--pagination-item-margin, 0 var(--spacer-2xs));
-    color: var(--c-text-muted);
     @include font(
       --pagination-font,
-      var(--font-normal),
-      var(--font-lg),
+      var(--font-weight--medium),
+      var(--font-size--lg),
       1.4,
-      var(--font-family-secondary)
-    );
+      var(--font-family--secondary)
+    );  
     &--prev, &--next {
       --pagination-item-width: 1.125rem;
-      --pagination-item-height: 1.125rem;
+      --pagination-item-height: 18px;
```

Comment:
So one dimension should be in rem and the other in px?

Reply:
of course not :)